cli/tests/unit_node/perf_hooks_test.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import * as perfHooks from "../../../ext/node/polyfills/perf_hooks.ts";
+import { performance } from "../../../ext/node/polyfills/perf_hooks.ts";
```

Comment:
Please import these from `node:perf_hooks` specifier, which are built-in Node compat APIs